packages/components/src/Stepper/Stepper.component.js

```diff
@@ -170,6 +172,16 @@ function Stepper({ steps, title, renderActions, children }) {
 		}
 	}, [steps]);
 
+	if (isInError) {
```

Comment:
The isInError is still used after, could you remove the usage on the code after this if branch ?

Reply:
Yes 👍🏼